civitai-downloader-v2/src/core/security/scanner.py

```
@@ -0,0 +1,744 @@
+#!/usr/bin/env python3
+"""
+Security Scanner for CivitAI Downloader.
+Provides malware detection, file validation, and security scanning capabilities.
+"""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)